spec/controllers/events_controller_spec.rb

```diff
@@ -1,29 +1,165 @@
 require 'rails_helper'
 
 RSpec.describe EventsController, type: :controller do
-  describe 'POST events#create' do
-    let!(:admin) { FactoryBot.create(:admin) }
+  let(:admin) { create(:admin) }
+  let(:user) { create(:user) }
 
-    before(:each) do
-      login_user admin
+  let(:event_params) do
```

Comment:
Move `event_params` to `POST :create` block and last test should use event data.

Reply:
`let` - moved to `Post: create`. Also, was replaced event_params to event data in last test case
@VasylYaremkiv